doxy-my-docs .py

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env python
```

Comment:
Please remove trailing space at the end of file name

Reply:
Fixed